misc/tracegen.py

```diff
@@ -7,12 +7,13 @@
 import os
```

Comment:
Should we just kill this in favour of https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/tracegen?

Reply:
It's nice to be able to load up a database without having to spin up both the open telemetry collector and the promscale connector. I like having this option for the sake of simplicity, and at this point, there's very little upkeep to it.  I don't see much benefit in killing it unless it becomes a big burden.

Reply:
You should able to tracegen without otel collector, but promscale is needed.
```
tracegen -otlp-endpoint localhost:9202 -otlp-insecure -rate 10000 -duration 10s
```

Since this tool bypasses promscale, I would recommend to move it to extension repo.